README.md

```diff
@@ -1458,143 +1458,53 @@ picovoice.delete();
+Install the Web SDK using yarn:
 
-Each spoken language is available as a dedicated npm package (e.g. @picovoice/picovoice-web-en-worker). These packages can be used with the @picovoice/web-voice-processor. They can also be used with the Angular, React, and Vue bindings, which abstract and hide the web worker communication details.
+```console
+yarn add @picovoice/picovoice-web
+```
 
-#### Vanilla JavaScript and HTML (CDN Script Tag)
+or using npm:
 
-```html
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <script src="https://unpkg.com/@picovoice/picovoice-web-en-worker/dist/iife/index.js"></script>
-    <script src="https://unpkg.com/@picovoice/web-voice-processor/dist/iife/index.js"></script>
-    <script type="application/javascript">
-      const RHINO_CONTEXT_BASE64 = /* Base64 representation of Rhino .rhn file */;
-
-      async function startPicovoice() {
-        console.log("Picovoice is loading. Please wait...");
-        picovoiceWorker = await PicovoiceWebEnWorker.PicovoiceWorkerFactory.create(
-          {
-            porcupineKeyword: { builtin: "Picovoice" },
-            rhinoContext: { base64: RHINO_CONTEXT_BASE64 },
-            start: true,
-          }
-        );
-
-        console.log("Picovoice worker ready!");
-
-        picovoiceWorker.onmessage = (msg) => {
-          switch (msg.data.command) {
-            case "ppn-keyword": {
-              console.log(
-                "Wake word detected: " + JSON.stringify(msg.data.keywordLabel)
-              );
-              break;
-            }
-            case "rhn-inference":
-              {
-                console.log(
-                  "Inference detected: " + JSON.stringify(msg.data.inference)
-                );
-                break;
-              }
+```console
+npm install --save @picovoice/picovoice-web
+```
 
-              writeMessage(msg);
-          }
-        };
+Create an instance of the engine using `PicovoiceWorker` and run on an audio input stream:
 
-        console.log("WebVoiceProcessor initializing. Microphone permissions requested ...");
+```typescript
+import { PicovoiceWorker } from "@picovoice/picovoice-web";
 
-        try {
-          let webVp = await WebVoiceProcessor.WebVoiceProcessor.init({
-            engines: [picovoiceWorker],
-            start: true,
-          });
-          console.log("WebVoiceProcessor ready! Say 'Picovoice' to start the interaction.");
-        } catch (e) {
-          console.log("WebVoiceProcessor failed to initialize: " + e);
-        }
-      }
+function wakeWordCallback(detection: PorcupineDetection) {
```

Comment:
for other bindings, we usually leave a "template-like" comment on how to use these callbacks, especially `inferenceCallback`.